test/spark/docker-compose.yaml

```diff
@@ -0,0 +1,88 @@
+version: '3.9'
+services:
+  postgres:
+    image: "postgres:11"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: lakefs
+      POSTGRES_PASSWORD: lakefs
+  lakefs:
+    image: "${REPO}/lakefs:${TAG}"
+    ports:
+      - "8000:8000"
+    networks:
+      default:
+        ipv4_address: 10.5.0.55
+    depends_on:
+      - "postgres"
+    volumes:
+      - ./lakectl-tester.yaml:/home/lakefs/.lakectl.yaml
+      - .:/local
+    environment:
+      - LAKEFS_GATEWAYS_S3_DOMAIN_NAME=s3.docker.lakefs.io
+      - LAKEFS_AUTH_ENCRYPT_SECRET_KEY=some random secret string
+      - LAKEFS_DATABASE_CONNECTION_STRING=postgres://lakefs:lakefs@postgres/postgres?sslmode=disable
+      - LAKEFS_BLOCKSTORE_LOCAL_PATH=/home/lakefs
```

Comment:
Needed?

Reply:
it makes my life easier to run this docker-compose locally and just set the storage type